code/modules/zombie/organs.dm

```diff
@@ -0,0 +1,87 @@
+#define START_TIMER reanimation_timer = world.time + rand(600,1200)
+
+/obj/item/organ/internal/zombie_infection
+	name = "festering ooze"
+	desc = "A black web of pus and vicera."
+	parent_organ = "head"
+	slot = "zombie_infection"
+	icon_state = "blacktumor"
+	var/causes_damage = TRUE
+	var/reanimation_timer
+	var/datum/species/old_species = /datum/species/human
+	var/living_transformation_time = 5
+	var/converts_living = FALSE
+
+/obj/item/organ/internal/zombie_infection/New()
+	. = ..()
+	if(iscarbon(loc))
```

Comment:
Maybe `if(ishuman(loc))` instead of the carbon check?

Reply:
If it affects vox I see no reason why not to let it affect IPC

Reply:
I think the problem is _how_ is it supposed to  affect them. It is a virus after all. IPCs don't get viruses

Reply:
It implants an organ into the target so in theory it should infect IPC as well

Reply:
No it shouldn't. IPC aren't organic at all. Vox are organic to a degree. IPC shouldn't be affected just as they can't be infected by xenos or terrors.

Reply:
And I doubt you can make enough argument that the virus makes an overgrowth that forces the body to move. Then you're just saying that the virus can make its own body and not need to infect someone, but it still does. The virus can't work with IPCs